vmdb/product/toolbars/servicetemplates-catalogs_center_tb.yaml

```diff
@@ -4,17 +4,7 @@
 ---
 :model: ServiceTemplate
 :button_groups:
-- :name: svc_catalog_lifecycle
+- :name: blank_buttons
```

Comment:
@skateman why is this being renamed to blank_button, you should just remove this toolbar yaml since that was the only button in this file and change application_helper to not load this toolbar yaml.